metadata.yaml

```diff
@@ -19,14 +19,31 @@ tags:
     - monitoring
     - alerting
     - grafana
-series:
-    - kubernetes
+bases:
+    - name: ubuntu
+      channel: 20.04/stable
```

Comment:
Is this needed? According to the [docs](https://discourse.charmhub.io/t/charm-metadata-v2/3674):
1. `series` is replaced by `containers/base`
2. `bases` is under `containers`, and "containers can include a single resource name **or** a list of bases."
2. there's a `bases` in `/manifest.yaml` (generated by charmcraft)

Reply:
There was a bug or actually missing feature in charmcraft because of which the charm would fail to build if bases was not listed in metadata.yaml, when I implemented this code. I can check again and remove this, if latest **stable** charmcraft does not have this issue.

Reply:
Just removed `bases` built and deployed charm. There was no error. So the issue seems to have been fixed. Pushing a commit now with bases removed.